test/parallel/test-debug-uncaught-exception-async.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const spawn = require('child_process').spawn;
+
+const emitUncaught = path.join(common.fixturesDir, 'debug-uncaught-async.js');
+const result = spawn(process.execPath, [emitUncaught], {encoding: 'utf8'});
+
+var stderr = '';
+result.stderr.on('data', (data) => {
+  stderr += data;
+});
+
+result.on('close', (code) => {
+  const expectedMessage =
+    "There was an internal error in Node's debugger. Please report this bug.";
```

Comment:
I used double-quotes to avoid escaping the apostrophe in `Node's`. (The linter allows for double-quotes if it avoids escaping a `'` character.) I can switch to single-quotes if that's deemed an inadequate reason to use double-quotes.